_benchmark/reference/workloads/test-procedures.md

```diff
@@ -0,0 +1,193 @@
+      "clients": 4,
+      "warmup-iterations": 1000,
+      "iterations": 1000,
+      "target-throughput": 100
+    }
+  ]
+}
+```
+
+#### Defining operations inline
+
+If you don't want reuse an operation in the schedule, you can also define operations inside the `schedule` element, as shown in the following example:
+
+```yml
+{
+  "schedule": [
+    {
+      "operation": {
+        "name": "force-merge",
+        "operation-type": "force-merge"
+      },
+      "clients": 1
+    },
+    {
+      "operation": {
+        "name": "match-all-query",
+        "operation-type": "search",
+        "body": {
+          "query": {
+            "match_all": {}
+          }
+        }
+      },
+      "clients": 4,
+      "warmup-iterations": 1000,
+      "iterations": 1000,
+      "target-throughput": 100
+    }
+  ]
+}
+```
+
+### Task options
+
+Each task contains the following options.
+
+Parameter | Required | Type | Description
+:--- | :--- | :--- | :---
+`operation` | Yes | List | Refers to either the name of an operation, defined in the `operations` element, or includes the entire operation inline.
+`name` | No | String | Specifies a unique name for the task when multiple tasks use the same operation.
+`tags` | No | String | Unique identifiers that can be used to filter between tasks.clients (optional, defaults to 1): The number of clients that should execute a task concurrently.
+`clients` | No | Integer | The number of clients that concurrently run the task. Default is `1`.
+
+### Target options
+
+OpenSearch Benchmark requires one of the following options when running a task.
+
+`target-throughput` | No | Integer | Defines the benchmark mode. When not defined, OpenSearch Benchmark assumes that this is a throughput benchmark and runs the task as fast as possible. This useful batch operations, where it is more important to achieve the best throughput as opposed to better latency. When defined, the target specifies the number of requests per second over all clients. For example, if you specify `target-throughput: 1000` with 8 clients, it means that each client will issue 125 (= 1000 / 8) requests per second. 
+`target-interval` | No | Interval | Defines an internal of less 1 / target-throughput (in seconds) less than one operation per second. Define either target-throughput or target-interval but not both (otherwise Rally will raise an error).
+`ignore-response-error-level` | No | Boolean | Controls whether to ignore errors encountered during the task when a benchmark is run with the `on-error=abort` command flag. 
+
+### Parallel tasks
+
+The `parallel` element runs tasks wrapped inside the element concurrently. 
+
+When running tasks in parallel, each task requires the `client` option to make sure clients inside your benchmark are reserved for that task. Otherwise, when the client option is specified inside the parallel element without a connection to the task, the benchmark will use that number of clients for all tasks.
```

Comment:
`When running tasks in parallel, each task requires the "client" option to make sure clients inside your benchmark are reserved for that task. Otherwise, when the client option is specified inside the parallel element without a connection to the task, the benchmark will use that number of clients for all tasks.` -> `When running operations in parallel, each operation requires the "client" option to make sure clients inside your benchmark are reserved for that operation. Otherwise, when the client option is specified inside the parallel element without a connection to an operation, the benchmark will use that number of clients for all operations.`